src/views/pages/reports/reports-page.js

```diff
@@ -21,16 +26,30 @@ export class ReportsPage extends Component {
       users: Map(),
       usersWhoDidntBuy: Map(),
       query: [],
+      addedEmailsStr:'',
+      validEmails: [],
+      invalidEmails: [],
```

Comment:
I'm not sure we need that, any invalid email should be tested individually and marked as invalid